Sources/Basic/OutputByteStream.swift

```diff
@@ -59,13 +59,13 @@ public class OutputByteStream: OutputStream {
     ///
     /// This method implicitly flushes the stream.
     public var bytes: ByteString {
-        flush()
+        try? flush()
         return ByteString(self.buffer)
     }
     
     // MARK: Data Output API
 
-    public func flush() {
+    public func flush() throws {
         // Do nothing.
```

Comment:
I'm not sure this should throw. If it throws, then it probably shouldn't be swallowed (e.g., in the `bytes` accessor` above).